social-distancing/libs/centroid_object_tracker.py

```diff
@@ -0,0 +1,71 @@
+# import the necessary packages
+from scipy.spatial import distance as dist
+from collections import OrderedDict
+import numpy as np
+
+class CentroidTracker():
```

Comment:
I suggest using a consistent naming convention. We like to use python black to lint the codebase.